cmd/alias.go

```diff
@@ -194,7 +194,7 @@ func (c *AliasCommand) Run(args []string) int {
 	spinner.Stop()
 	c.UI.Info("")
 	message := `
-Action Required: use the generated config by adding these lines to your wp-cli.yml or wp-cli.local.yml config.
+Action Required: use the generated config by adding these lines to site/wp-cli.yml or an alternative wp-cli.yml/wp-cli.local.yml config. 
```

Comment:
I like the `site/` addition, but `wp-cli.yml/wp-cli.local.yml` could be confusing because the `/` there is for "slash" and not a path separator right?

Maybe say "wp-cli.yml (or wp-cli.local.yml)" ?